src/main/scala/com/github/ahnfelt/react4s/Component.scala

```diff
@@ -90,7 +90,7 @@ sealed trait ElementOrComponent extends Tag {
 }
 
 /** Wraps a React component written in JavaScript. Example: ```DynamicComponent(js.Dynamic.global.MyJsComponent, js.Dictionary("label" -> "Go!"))``` */
-final case class DynamicComponent(componentClass : Any, props : Any, key : Option[String] = None, ref : Option[Any => Unit] = None) extends ElementOrComponent {
+final case class DynamicComponent(componentClass : Any, props : Any, children : Seq[Tag], key : Option[String] = None, ref : Option[Any => Unit] = None) extends ElementOrComponent {
```

Comment:
I'd kinda like to have empty defaults á la `props : js.Dictionary = js.Dicitionary.empy` and `children : Seq[Tag] = Seq.empty` here, but didn't dare to pull in `js.*`.